tensorflow/contrib/rnn/python/ops/lstm_ops.py

```diff
@@ -623,6 +640,25 @@ def __init__(self,
     # Inputs must be 3-dimensional.
     self.input_spec = base_layer.InputSpec(ndim=3)
 
+    # Dropout parameter.
+    class DummyRNNCell(object):
```

Comment:
i really don't like this approach of creating a dummy RNNCell just to get dropout.

how about refactoring the DropoutWrapper so that you can use its underlying helper functions here?